django_freeradius/tests/base/test_api.py

```diff
@@ -313,6 +313,61 @@ def test_accounting_list_200(self):
         self.assertEqual(item['nas_ip_address'], '172.16.64.91')
         self.assertEqual(item['calling_station_id'], '5c:7d:c1:72:a7:3b')
 
+    def test_accounting_filter_200(self):
+        data1 = self.acct_post_data
+        data1.update(dict(username='test_user',
+                          unique_id='75058e50'))
+        self.radius_accounting_model.objects.create(**data1)
+        data2 = self.acct_post_data
+        data2.update(dict(called_station_id='E0-CA-40-EE-D1-0D',
+                          unique_id='99144d60'))
+        self.radius_accounting_model.objects.create(**data2)
+        data3 = self.acct_post_data
+        data3.update(dict(calling_station_id='4c:8d:c2:80:a7:4c',
+                          unique_id='12144d40'))
+        self.radius_accounting_model.objects.create(**data3)
+        data4 = self.acct_post_data
+        data4.update(dict(start_time='2018-03-02T00:43:24.020460+01:00',
+                          unique_id='22145d44'))
+        self.radius_accounting_model.objects.create(**data4)
+        data5 = self.acct_post_data
+        data5.update(dict(stop_time='2018-03-02T00:43:24.020460+01:00',
+                          unique_id='93184d40'))
+        self.radius_accounting_model.objects.create(**data5)
+        data6 = self.acct_post_data
+        data6.update(dict(stop_time=None,
+                          unique_id='62144d40'))
+        self.radius_accounting_model.objects.create(**data6)
+        response = self.client.get('{0}?username=test_user'.format(self._acct_url))
+        self.assertEqual(len(response.json()), 1)
+        self.assertEqual(response.status_code, 200)
+        item = response.data[0]
+        self.assertEqual(item['username'], 'test_user')
+        response = self.client.get('{0}?called_station_id=E0-CA-40-EE-D1-0D'.format(self._acct_url))
+        self.assertEqual(len(response.json()), 1)
+        self.assertEqual(response.status_code, 200)
+        item = response.data[0]
+        self.assertEqual(item['called_station_id'], 'E0-CA-40-EE-D1-0D')
+        response = self.client.get('{0}?calling_station_id=4c:8d:c2:80:a7:4c'.format(self._acct_url))
+        self.assertEqual(len(response.json()), 1)
+        self.assertEqual(response.status_code, 200)
+        item = response.data[0]
+        self.assertEqual(item['calling_station_id'], '4c:8d:c2:80:a7:4c')
+        response = self.client.get('{0}?start_time=2018-03-02T00:43:24.020460+01:00'.format(self._acct_url))
```

Comment:
we should allow users (developers) to filter by a date and not a datetime.
That is, we want to filter by the entire date, eg: `2018-03-02` and not `2018-03-02T00:43:24.020460+01:00`

But since you will be freezing the date on top with `@freeze_time`, use `START_DATE` as follows:
```python
response = self.client.get('{0}?start_time={1}'.format(self._acct_url, START_DATE))
```
I just tried it and it seems to pass this assertion.